vignettes/adsl.Rmd

```diff
@@ -677,16 +677,45 @@ dataset_vignette(
 
 Numeric and categorical variables (`AGE`, `RACE`, `COUNTRY`, etc.) may need to be grouped to perform
 the required analysis.
-`{admiral}` does not **currently** have functionality to assist with all required groupings. So, the user will often need to create his/her own function to meet his/her study requirement. 
+`{admiral}` provides the `derive_vars_cat()` function to create such groups.
+This function is especially useful if more than one variables need to be created for
+each condition, which is not the case here.
```

Comment:
```suggestion
This function is especially useful if more than one variable needs to be created for
each condition, e.g., `AGEGR1` and `AGEGR1N.
```
Below I would use `derive_vars_cat()` for deriving `AGEGR1` and `AGEGR1N` and `mutate()` for `REGION1`. I think this makes it more concise.